### ISSUE TITLE
Install method should mention building assets

### ISSUE THREAD

**Issue by battosai30** [alabs/decidim-module-calendar#22]
Hi,

Installation guide should mention that you have to build assets with :
`bin/rails assets:precompile db:migrate RAILS_ENV=production`
(I am not used at all to Ruby so I just understand that this command build assets in my production env ...)

If you do not, plugin is not working (internal server error)

**Comment by mijailr** [comment]
Hi @battosai30, the `assets:precompile` and `db:migrate` step is documented as part of the decidim and most of the rails applications. You can refer to the documentation on decidim's repository: 

https://github.com/decidim/decidim/blob/develop/docs/manual-installation.md

**Comment by battosai30** [comment]
Yes I agree, I just wanted to notice that a mention that you need to precompile assets is needed to complete install of calendar module.

As I said I am a noob about Ruby, but not about sysadmin. As I lost some time finding what to do, I think a simple modification of installation guide would be welcome :)

**Comment by MohammedSalahadin** [comment]
Hi @battosai30
Actually I would like to thank you because the command that you have posted worked for me. without it it wont work. 
Cheers
